lib/src/models/result_movie.dart

```diff
@@ -1,10 +1,10 @@
 import '../utils/movie_strings.dart';
 
 class ResultMovie {
-  int? voteCount;
+  late int voteCount;
   late int id;
   bool? video;
-  var voteAverage;
+  dynamic voteAverage;
```

Comment:
why voteAverage is dynamic? You can parse the value in ResultMovie constructor

Reply:
I think it is a dynamic type, because when I use it to show voteAverage I need it to be String type and when I need to do the logic to show the list of stars I need it to be Integer type so, what do you think that it would be better in this case? Maybe left it just in Integer? Or dynamic as I thought or var as it was?

Reply:
Whats the data type thats the server is sending in the response ?

Reply:
In that case, the type on json said that is integer, if you need to change it to String, you can parse it.

Reply:
exactly !

Reply:
The type of data that server returns is 'dynamic', that's why I also changed into it

Reply:
Fixed, I finally lefted var type.